streamparse/__init__.py

```diff
@@ -6,14 +6,12 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-import streamparse.base
-import streamparse.bolt
+import streamparse.storm
 import streamparse.cmdln
 import streamparse.contextmanagers
 import streamparse.debug
 import streamparse.decorators
-import streamparse.ipc
-import streamparse.spout
+from streamparse.storm import bolt, component, spout
```

Comment:
I did this to try to not break the API for most users (although the rare person directly referencing `streamparse.ipc` will be out of luck).